app/src/Client/ClientApi.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace Client;
+
+use Application\BaseApi;
+use User\UserApi;
+
+class ClientApi extends BaseApi
+{
+    /**
+     * Get all clients associated with the current user
+     *
+     * @return array
+     */
+    public function getCollection($queryParams)
+    {
+        $talks_uri = $this->baseApiUrl . '/v2.1/applications';
+
+        $talks = (array) json_decode(
```

Comment:
What about passing the `json_decode` second argument `true` so it returns an array rather than an object?